app/src/main/java/org/sopt/official/di/attendance/AttendanceBindsModule.kt

```diff
@@ -0,0 +1,17 @@
+package org.sopt.official.di.attendance
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import org.sopt.official.data.repository.attendance.AttendanceRepositoryImpl
+import org.sopt.official.domain.repository.attendance.AttendanceRepository
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class AttendanceBindsModule {
+    @Binds
+    @Singleton
+    abstract fun bindAttendanceRepository(attendanceRepositoryImpl: AttendanceRepositoryImpl): AttendanceRepository
+}
```

Comment:
굳이 모듈 두개로 안나눠도 하나에서 다 선언할 수 있습니다. 아래 코드로 테스트 해봤는데 무리 없이 앱 실행되는 것 확인했습니다.

```suggestion
@Module
@InstallIn(SingletonComponent::class)
abstract class AttendanceModule {
    @Binds
    @Singleton
    abstract fun bindAttendanceRepository(
        attendanceRepositoryImpl: AttendanceRepositoryImpl
    ): AttendanceRepository

    companion object {
        @Provides
        @Singleton
        fun provideAttendanceService(): AttendanceService = MockAttendanceService()
    }
}
```

Reply:
확인 후 적용해보겠습니다. :-)